src/ansible_creator/resources/collection_project/plugins/modules/hello_world.py.j2

```diff
@@ -0,0 +1,22 @@
+# GNU General Public License v3.0+
+
+DOCUMENTATION = """
+    module: hello_world
+    author: Ansible Core Team
+    version_added: "1.0.0"
+    short_description: A custom action plugin for Ansible.
+    description:
+      - This is a custom action plugin to provide action functionality.
+    notes:
+      - This is a scaffold template. Customize the plugin to fit your needs.
+"""
+
+EXAMPLES = """
+- name: Example Action Plugin
+  hosts: localhost
+  tasks:
+    - name: Example hello_world plugin
+      with_prefix:
```

Comment:
can we add the with_prefix, prefix and msg keys to the doc string so it matches the example?

and use these keys in the logic of the plugin.

We should also show some argspec validation, it's been a while but I don't think it happens automatically with an action plugin. In the past we had to do it like this https://github.com/ansible-collections/ansible.utils/blob/dda46ee3cf32acb0b63f23451c56ba09f03eb456/plugins/action/fact_diff.py#L34  but there may be a better way now

Reply:
Done!